src/commands.ts

```diff
@@ -185,6 +187,39 @@ export async function addNewClass(classPath: string, classType: string) {
   console.log("\x1b[36m%s\x1b[0m", "Class added successfully.");
 }
 
+export async function deleteProjectHandler(projectId : string, forced : boolean) {
+  // show prompt if no project id is selected
+  if (typeof projectId === 'string' && projectId.trim().length === 0) {
+    console.log('No project ID specified, select an ID to delete from this list:')
+    const projects = await listProjects();
+    if (projects.length === 0) {
+      return false;
+    }
+
+    const selection = await askQuestion(`Please select project number to delete (0--${projects.length - 1}) [none]: `, "");
+    const selectionNum = Number(selection);
+    if (isNaN(selectionNum) || selectionNum < 0 || selectionNum >= projects.length) {
+      console.log("No valid selection was made, aborting.");
+      return false;
+    } else {
+      forced = false;
+      projectId = projects[selectionNum][2].split(':')[1].trim();
+    }
+  }
+
+  if (!forced) {
+    const confirmation = await askQuestion(`Are you sure you want to delete project ${projectId}? [Y]/n: `, "n");
```

Comment:
default value is "Y" or "n". In the message is marked as Y but the implementation is with "n"